journey/pepr-upgrade.test.ts

```diff
@@ -97,7 +99,8 @@ describe("Should test Pepr upgrade", peprUpgrade);
 
 function getManifestData(): { manifestUUID: string; matchedFile: string } {
   const moduleDirectory = path.join("./pepr-upgrade-test", "dist");
-  const manifestPattern = /.*pepr-module-([a-f0-9-]+)\.yaml$/;
+  // https://regex101.com/r/URNKAg/1
```

Comment:
This regex did not appear to match when I checked the link. Is that expected? 
<img width="986" alt="Screenshot 2025-03-24 at 10 19 06" src="https://github.com/user-attachments/assets/80787dea-11f0-4757-bf50-68b349fc2da8" />

Reply:
ah the extra return burned me

Reply:
they should all match now